### PR TITLE
add Community structure

### PR DESCRIPTION
**Pull request by jude-zhu**
as titled

### CODE REVIEW

File: community/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Nebula Graph Community Architecture
+
+The Nebula Graph community is organized as shown below:
+
+![Nebula Community Architecture](./images/structure.png)
+
+## PMC
+
+PMC (Project Management Committee) is the entity the controls and leads the whole nebula project. PMC members have the right to propose a new Maintainers or Committers, and the right to vote for the major decisions for Nebula Graph. See [PMC List](./pmc-list.md) for a list of PMC members.
```
Comment (oldLady344) [inline community/README.md:9]: is the entity that controls and leads the whole nebula project. 

PMC members have the right to propose a new Maintainer or Committer

File: community/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+
+## Maintainer
+
+The planner and designer of the repository, with the right to merge branches into the master. A Maintainer should:
```
Comment (oldLady344) [inline community/README.md:14]: A Maintainer is a planner and designer ....?

File: community/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Recommended by Maintainer or Project Management Committee (PMC) as a Committer who has made a significant contribution to the repository. A committer has approval permission for code reviews of the repository. See [Committer List](./committer-list.md) for the Committers of each repository.
+
+> Note: Each repository requires at least 2 approvals for each PR to be merged into the master branch.
+
+### How to become a Committer of a Repository
+
+- Generated from Active Contributors.
+- Has more than 5 PRs merged to the master branch of the repository within a year.
+- Nominated by the Matainter of the repository or PMC and gained majority(1/2) votes from the Decision-Making Group(consists of Maintainers of the repository and the PMC).
```
Comment (oldLady344) [inline community/README.md:37]: Nominated by a Maintainer (? 是只要一个Maintainer就行还是得所有 Maintainers?) of the repository or the PMC and gained the majority (1/2) votes from the Decision-Making Group, which consists of the Maintainers of the repository and the PMC.

File: community/README.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+- The appointment is for one year.
+
+
+## Active Contributor
+
+Active Contributors are continuously active contributors in the community. They can have issues and PRs assigned to them, participate in the development. See [Active Contributor List](active-contributor-list.md) for the list of Active Contributors.
```
Comment (oldLady344) [inline community/README.md:43]: and participate in development.

File: community/README.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+### How to become an Active Contributor of a Repository
+
+- Contributes at least 5 PRs to a specific repository within one year will become an active contributor automatically.
```
Comment (oldLady344) [inline community/README.md:47]: If you contribute at least 5 PRs to a specific repository within one year, you will become an ....

File: community/README.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
+# Nebula Graph Community Architecture
+
+The Nebula Graph community is organized as shown below:
```
Comment (oldLady344) [inline community/README.md:3]: as shown below.

**Comment by jude-zhu** [comment]
> Shall we add these docs in the doc index file?
> https://github.com/vesoft-inc/nebula-docs/blob/master/mkdocs.yml
> https://github.com/vesoft-inc/nebula-docs/blob/master/docs/manual-EN/README.md

todo. After this pr is reviewed (*￣︶￣)

File: community/README.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+- Generated from Active Contributors
+- Has more than 5 PRs merged to the master branch of the repository within a year
+- Self-recommended or Nominated by a Maintainer of the repository or PMC
+- Gain majority (1/2) votes from the Decision-Making Group (consists of Maintainers of the repository and the PMC)
+- The appointment is for one year
+
+
+### Active Contributor
+
+Active Contributors are continuously active contributors in the community. They can have issues and PRs assigned to them, participate in development. See [Active Contributor List](active-contributor-list.md) for the list of Active Contributors.
```
Comment (oldLady344) [inline community/README.md:46]: and participate in development.

File: community/README.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+The PMC (Project Management Committee) is the entity that controls and leads the whole Nebula Graph projects. 
+PMC members are responsible for voting in new maintainers or committers, and the authority to make all major decisions for Nebula Graph. See [PMC List](./pmc-list.md) for the list of PMC members.
+
+
+### Maintainer
+
+Maintainers are the planner and designer of the repository, with the right to merge branches into the master. A Maintainer should:
+
+- Set technical directions, roadmap, and priorities for the repository
+- Drive the development forward and ensure newcomers, as well as long-time contributors have a great experience
```
[the remark this answers is not in the document]
Comment (jude-zhu) [reply community/README.md:18]: refer to https://dictionary.cambridge.org/dictionary/english/experience
"have a great dining experience"

File: community/README.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+- If you contribute at least 5 PRs to a specific repository within one year, you will become an active contributor automatically
+
+### Contributor
+
+Anyone who contributes one PR for any repository is a Contributor.
+
+#### How to become a Contributor
+
+- contributes at least 1 PR for any project in the [vesoft-inc organization](https://github.com/vesoft-inc)
```
[the remark this answers is not in the document]
Comment (jude-zhu) [reply community/README.md:58]: Good catch

File: community/README.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
+## Nebula Graph Community Architecture
+
+The Nebula Graph community is organized as shown below.
+
+![Nebula Community Architecture](./images/structure.png)
+
+### PMC
+
+The PMC (Project Management Committee) is the entity that controls and leads the whole Nebula Graph projects. 
+PMC members are responsible for voting in new maintainers or committers, and the authority to make all major decisions for Nebula Graph. See [PMC List](./pmc-list.md) for the list of PMC members.
```
[the remark this answers is not in the document]
Comment (jude-zhu) [reply community/README.md:10]: a bit confused about "vote in" & "vote for".  "vote for" as I understand, it the opposite of "vote against".
https://dictionary.cambridge.org/dictionary/english/vote

**Pull request merged**